src/main/content/collection-management.html

```diff
@@ -4,47 +4,56 @@
+            <hr/>
         </div>
     </div>
-</div>
 
-<div class="container">
+    <!-- Intro text -->
     <div class="row">
-        <div class="col centeredCol">
+        <div class="col text-center">
             <div>
                 <p>
-                        The solution architect (or someone serving in that role) provides governance over the set of collections available for developers to seamlessly use, empowering developers with agility and speed while complying with company policies. This management CLI andaccess to the correct Github repositories enable this governance. 
+                    The solution architect (or someone serving that role) provides governance over the set of collections available for developers to seamlessly use, 
+                    empowering developers with agility and speed while complying with company policies. This management CLI and access to the correct GitHub repositories 
+                    enable this governance. 
                 </p>
                 <p id="introText">
-                        Learn more by checking out our <a href="https://github.com/kabanero-io/kabanero-command-line/tree/master/kab-cli/kabanero-mgmt">Kabanero github pages</a>. 
+                    Learn more by checking out our 
+                    <a href="https://github.com/kabanero-io/kabanero-command-line/tree/master/kab-cli/kabanero-mgmt" target="_blank">Kabanero GitHub pages</a>. 
                 </p>                 
             </div>
+            <hr/>
         </div>
     </div>
-</div>
 
+    <!-- Try Kabanero using the CLI title -->
+    <div class="row">
+        <div class="col text-center">
+            <h2>Try Kabanero using the CLI</h2><br>
+            <p>You will need a Kabanero instance installed to connect our CLI to. 
+                <a href="/operations/kabanero-foundation-setup/"> Check out the Kabanero Foundation Try it experience for more details.</a>
+            </p>
+        </div>
+    </div>
 
-<div class="container">
-    <div class="row horizontal-separator bottomMargin"></div>
+    <!-- Try Kabanero using the CLI Steps -->
     <div class="row">
-        <div class="col">
-            <h2>Try it using the CLI</h2><br>
-            <p>You will need a Kabanero instance installed to connect our CLI to.  Check out the Kabanero Foundation Try it experience for more details.</p>
+        <div class="col-md-4"></div>
+        <div class="col-md-4">
             <ol>
                 <li>
-                    <a href="https://github.com/kabanero-io/kabanero-command-line/tree/master/kab-cli/kabanero">Install the CLI on your laptop</a>
+                    <a href="https://github.com/kabanero-io/kabanero-command-line/tree/master/kab-cli/kabanero">Install the CLI</a>
                 </li>
                 <li>
                     Activate our first collection
                 </li>
             </ol>
         </div>
+        <div class="col-md-4"></div>
     </div>
```

Comment:
The doc PR for collections got merged (https://github.com/kabanero-io/docs/pull/21) so this page needs to consume that doc

This section needs a liquid script similar to the one in the foundation setup doc: 
```
{% assign collectionsDoc = site.pages | where: 'title', 'Kabanero Collections' | sort: 'title' %}

{% for doc in collectionsDoc limit:1 %}
    <h3>{{doc.title}}</h3>
    {{ doc.content }}
{% endfor %}
```